### DISCUSSION TITLE
Option to default to collapsed side-bar table of content

### DISCUSSION THREAD

**Discussion by kingfungdesign** [outline/outline#4973] [Ideas]
Problem:
The side-bar table of contents is defaulted to expand all layers for publicly shared pages, making the sidebar very long and hard to navigate for large set of documents.

One possible solution is to simply provide the option to default with everything collapsed.

**Comment by sarahjcotton** [comment]
Just adding that wherever there is an option to expand a menu it would also be helpful to have a toggle to expand/collapse all, which is pretty standard afaik. When browsing for content as opposed to searching, having to close everything I've opened is rather tedious.
